scripts/run.sh

```diff
@@ -19,14 +19,14 @@ fi
 VERSION=$1
 if [[ -z "${VERSION}" ]]; then
   echo "Missing version argument!"
-  echo "Usage: ${0} [VERSION] [GENESIS_ADDRESS]" >> /dev/stderr
+  echo "Usage: ${0} [VERSION] [GENESIS_ADDRESS]" >>/dev/stderr
```

Comment:
Is there a reason for the removed spaces? If not, can we add them back in?

Reply:
no, seems my editor extension formatted it.